lib/appsignal/error_handler.ex

```diff
@@ -201,6 +201,9 @@ defmodule Appsignal.ErrorHandler do
     msg = Exception.message(r)
```

Comment:
the commit message of this commit should be "move" rather than "remove" I think?
Also the test you added earlier in the plug should also be added to the test for this module? (Or moved)